Rakefile

```diff
@@ -0,0 +1,15 @@
+require 'bundler/gem_tasks'
```

Comment:
looks good. I tested locally a couple of these built-in rake tasks and worked fine. 
`rake build`                                   
`rake clean`